perun-utils/ldapc-scripts/initialize-ldap.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/sh
+
+#Option test is only for generating diff between perun and LDAP without any action
+while getopts "t" opt; do
+    case "$opt" in
+    t)
+        TEST=1
+				echo "Initialization executed in the test mode - without any changes!"
+        ;;
+    esac
+done
+
+if [ ! ${TEST} ]; then
+  #If LDAPc is running, inform us and exit without processing
+  LDAPC_PID_FILE=/home/perun/perun-ldapc/pid
```

Comment:
In ldapc start/stop scripts you also check ``/proc/pid`` if ldapc process is actually running, it would be better to use it here too, since if killed, pid file remains.

Reply:
We agreed not to check it.